buildconfig/pygame-stubs/bufferproxy.pyi

```diff
@@ -1,11 +1,12 @@
-from typing import Any, overload
+from typing import Any, overload, Optional
+
 
 class BufferProxy(object):
     parent: Any
     length: int
-    raw: bytes
+    raw: str
```

Comment:
This is bytes in python3 and str in python2. pgcompat.h

Reply:
I had some problems with mypy where it couldn't accept it as AnyStr, but I will rewrite it as Union[bytes, str]